<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配置中心</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>项目目录：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_project">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>打开...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>90</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#ff007f;&quot;&gt;EasyTest&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#aa00ff;&quot;&gt;配置中心&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>测试方案目录：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>测试用例目录：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>参数化目录：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fangan">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>测试方案目录：111111111111111</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_testCase">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>测试方案目录：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_param">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>测试方案目录：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>161</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>1.测试方案</string>
   </property>
   <widget class="QListWidget" name="listWidget_testplan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>401</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_createtestplan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>创建方案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_removetestplan">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>移除方案</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_testcase">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>591</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>2.测试用例套件</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="checkBox_ff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Firefox</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_ch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Chrome</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_ie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>IE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ffremote">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>10</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chremote">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ieremote">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget_testCase">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>491</width>
      <height>301</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>启用</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>用例名称</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>用例路径</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>参数化路径</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_createtestcase">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加用例</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_down">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_removetestcase">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>移除</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>81</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_save">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>启动</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_fresh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
